.vscode/settings.json

```diff
@@ -5,6 +5,7 @@
     },
     "python.terminal.activateEnvInCurrentTerminal": true,
     "python.defaultInterpreterPath": "./dbt-env/bin/python3",
+    "dbt.dbtPythonPathOverride": "./dbt-env/bin/python3",
```

Comment:
Hah 😄 I should have scrolled here before responding to the previous comment. Agree with this change - the user should be responsible for managing the environment rather than making this part of the codebase. But, this needs to be discussed with the others as it will change the behaviour of vscode for users.

Reply:
so, i had set it up this way so that folks using VS Code would get their virtualenv activated each time they open the repo, without having to remember to activate it.  if there's a way to instruct people to set this up _without_ committing it to the repo, i'm open to doing that instead - is there?

Reply:
Setting up and activating an environment should be the user's responsibility and outside of source control. What if someone wants to use `uv` or `conda` to manage their environments or use something other than vscode for development?

I would propose moving this to the documentation in README and modifying  my devcontainer hack to get rid of setting up a virtual environment within a devcontainer - which as @lawrenceadams rightly pointed out, is redundant.

Reply:
In this case should we add .vscode to .gitignore?  (and then maybe we could add this settings json file to an extras folder so users can easily copy it to their local if they want to use it without thinking too much about it)

Reply:
My preference is to keep .vscode under source control to share settings (eg. spaces Vs tabs, task configuration, etc.) - there is a good discussion here - https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control

However, environment activation steps can simply go in README rather than in code. I would expect the user to be able to setup a virtual environment if they are planning to use DuckDb, Postgres and dbt. If not, good time to learn anyway.

I will update this PR (with rebase).

Reply:
Sounds good to me!  Please go ahead and include that change in this PR :)